.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -103,61 +105,105 @@ steps:
+          imagePrefix: "${AWS_IMAGE_UBUNTU_ARM_64}"
+          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "auditbeat: Linux ARM Integration Tests"
+
+      - label: ":linux: Auditbeat Linux ARM Unit Tests"
         key: "auditbeat-extended-arm64-unit-tests"
+        if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
         command: |
           set -euo pipefail
           cd auditbeat
-          mage unitTest
+          mage build unitTest
         agents:
           provider: "aws"
           imagePrefix: "${AWS_IMAGE_UBUNTU_ARM_64}"
           instanceType: "${AWS_ARM_INSTANCE_TYPE}"
-        artifact_paths: "auditbeat/build/*.*"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "metricbeat: Linux arm64 tests"
+              context: "auditbeat: Linux ARM Unit Tests"
 
-  - group: "Auditbeat MacOS Extended"
-    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
-    steps:
-      - label: ":mac: Auditbeat macOS x86_64 Unit Tests"
+      - label: ":mac: Auditbeat MacOS Unit Tests"
+        if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
         command: |
           set -euo pipefail
           source .buildkite/scripts/install_macos_tools.sh
           cd auditbeat
-          mage unitTest
+          mage build unitTest
         agents:
           provider: "orka"
           imagePrefix: "${IMAGE_MACOS_X86_64}"
-        artifact_paths: "auditbeat/build/*.*"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "auditbeat: Extended MacOS x86_64 Unit Tests"
+              context: "auditbeat: MacOS Unit Tests"
 
-      - label: ":mac: Auditbeat macOS arm64 Unit Tests"
+      - label: ":mac: Auditbeat MacOS ARM Unit Tests"
+        if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macOS.*/
         command: |
           set -euo pipefail
           source .buildkite/scripts/install_macos_tools.sh
           cd auditbeat
-          mage unitTest
+          mage build unitTest
         agents:
           provider: "orka"
           imagePrefix: "${IMAGE_MACOS_ARM}"
-        artifact_paths: "auditbeat/build/*.*"
+        artifact_paths:
+          - "auditbeat/build/*.xml"
+          - "auditbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "auditbeat: Extended MacOS arm64 Unit Tests"
+              context: "auditbeat: MacOS ARM Unit Tests"
 
   - group: "Auditbeat Windows Extended Testing"
```

Comment:
in the future we'll likely need to also explicitly say x86_64 or arm64 here, but let's keep it as is for now.